### PR TITLE
Update xdebug config

### PR DESCRIPTION
**Pull request by bertrand-leroux**
This PR aims at using xdebug3 config settings

### CODE REVIEW

**Comment by jmleroux** [comment]
We should also add a README to explain how to enable/disable xdebug

**Comment by bertrand-leroux** [comment]
done

File: php/7.4/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# ... 
+    environment:
+        PHP_IDE_CONFIG: 'serverName=your_app'
+        XDEBUG_CONFIG: >-
+            client_port=9003
+            client_host=172.17.0.1
+            discover_client_host=1
+            idekey=PHPSTORM
+        XDEBUG_MODE: '${XDEBUG_MODE:-off}'
+        XDEBUG_SESSION: 1
```
Comment (jmleroux) [inline php/7.4/README.md:19]: What is the XDEBUG_SESSION?

**Comment by jmleroux** [comment]
Closed in favor of #11

**Pull request closed without merging**